mwclient/__init__.py

```diff
@@ -24,7 +24,7 @@
 """
 
 from mwclient.errors import *  # noqa: F401, F403
-from mwclient.client import Site, __version__  # noqa: F401
+from mwclient.client import Site as Site, __version__ as __version__  # noqa: F401
```

Comment:
why?

Reply:
It's required by mypy due to `--no-implicit-reexport`, which is part of `--strict`. Without this we get warning such as the following:

```
test/test_client.py:90: error: Module "mwclient" does not explicitly export attribute "Site"  [attr-defined]
test/test_client.py:129: error: Module "mwclient" does not explicitly export attribute "__version__"  [attr-defined]
...
```

The alternative is to disable `--no-implicit-reexport` in the mypy config.